NEWS.md

```diff
@@ -1,5 +1,7 @@
 # testthat (development version)
 
+* Fix bug revealed by fix to another bug in cli (#1606).
+
```

Comment:
I am not sure if this is very useful to the end user, maybe we don't need to mention it at all, as nobody has really seen this bug?

Reply:
Yeah, not sure why I wrote that. I'll just add something generic about cli.